doc/source/serve/advanced.rst

```diff
@@ -421,3 +421,24 @@ in :mod:`serve.start <ray.serve.start>`:
    Using the "EveryNode" option, you can point a cloud load balancer to the
    instance group of Ray cluster to achieve high availability of Serve's HTTP
    proxies.
+
+Routing
```

Comment:
This section name may be confusing because users might think it refers to how requests are routed through serve or something like that. How about "Variable HTTP Routes" ?